Formula/apache-arrow-glib.rb

```diff
@@ -18,16 +18,24 @@ class ApacheArrowGlib < Formula
   end
 
   depends_on "gobject-introspection" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
   depends_on "pkg-config" => :build
   depends_on "apache-arrow"
   depends_on "glib"
 
   def install
+    args = std_meson_args + %W[
+      -Diconv=auto
+      -Dgio_module_dir=#{HOMEBREW_PREFIX}/lib/gio/modules
+      -Dbsymbolic_functions=false
+      -Ddtrace=false
+    ]
     cd "c_glib" do
-      system "./configure", "--prefix=#{prefix}"
-      system "make"
-      system "make", "install"
+      system "meson", *args, ".."
     end
+    system "ninja", "-v"
+    system "ninja", "install", "-v"
```

Comment:
How about creating a build directory instead of existing the top source directory?

```ruby
mkdir "build" do
  system "meson", *std_meson_args, "../c_glib"
  system "ninja", "-v"
  system "ninja", "install", "-v"
end
```